docs/seq-ssi/flip-flops.md

```
@@ -130,6 +130,30 @@ Next state of D flip-flop is always equal to data input, D for every positive tr
+```verilog
+// D Latch — level-sensitive (NOT what CircuitVerse implements)
+always @(D or en) begin
+    if (en) Q = D;
+end
+```
+
+> **Note:** Always use `<=` (non-blocking assignment) inside
+> `always @(posedge clk)` blocks. Using `=` (blocking assignment)
+> in clocked flip-flop logic is a common mistake that causes
+> simulation mismatches.
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)